ivi/rohdeschwarz/__init__.py

```diff
@@ -25,7 +24,7 @@
 
 """
 
-# RF Signal generators
-# SMC 100A
-from .rohdeschwarzSMC100A import rohdeschwarzSMC100A
-
+# Oscilloscopes
+# RTB2000 series
+from .rohdeschwarzRTB2002 import rohdeschwarzRTB2002
+from .rohdeschwarzRTB2004 import rohdeschwarzRTB2004
```

Comment:
@JonasLang, I dropped this newline while trying to merge the conflicts on the web gui.  Could you pleas add a newline character at the end of the line? (but not an empty line)